projects/1.18/assets/create-clockwork/vs_clockwork/lang/zh_cn.json

```
@@ -0,0 +1,129 @@
+  "item.vs_clockwork.combustion_engine": "内燃机",
+  "item.vs_clockwork.pastrymaker": "糕点师",
+  "block.vs_clockwork.ballooner": "气球",
+  "item.vs_clockwork.ballooner": "气球",
+  "block.vs_clockwork.alt_meter": "高度计",
+  "item.vs_clockwork.alt_meter": "高度计",
+  "block.vs_clockwork.afterblazer": "燃烧过后",
+  "item.vs_clockwork.afterblazer": "燃烧过后",
+  "block.vs_clockwork.intake": "进气",
+  "item.vs_clockwork.intake": "进气",
+  "block.vs_clockwork.ballast": "压舱物",
+  "item.vs_clockwork.ballast": "压舱物",
+  "block.vs_clockwork.goo_block": "黏性方块",
+  "item.vs_clockwork.goo_block": "黏性方块",
+  "block.vs_clockwork.slicker": "雨衣",
+  "item.vs_clockwork.slicker": "雨衣",
+
+  "block.vs_clockwork.wanderlite_deepslate_ore": "深层温德莱矿石",
+  "item.vs_clockwork.wanderlite_deepslate_ore": "深层温德莱矿石",
+  "block.vs_clockwork.wanderlite_end_ore": "末地温德莱矿石",
+  "item.vs_clockwork.wanderlite_end_ore": "末地温德莱矿石",
+  "item.vs_clockwork.wanderlite_crystal": "温德莱水晶",
+  "item.vs_clockwork.wanderlite_cube": "温德莱立方体",
+  "item.vs_clockwork.wanderlite_matrix": "温德莱矩阵",
+  "item.vs_clockwork.incomplete_wanderwand": "不完整的魔杖",
+
+
+  "block.vs_clockwork.heat_pipe": "热管",
+  "item.vs_clockwork.heat_pipe": "热管",
+  "item.vs_clockwork.stratodonut": "高空环流",
+  "item.vs_clockwork.physics_infuser": "物理注入器",
+  "item.vs_clockwork.wanderwand": "魔杖",
+  "vs_clockwork.subtitle.physinitialize": "物理注入器已初始化",
+  "vs_clockwork.subtitle.physfinish": "物理已注入",
+  "vs_clockwork.subtitle.zap": "Zap!",
+  "vs_clockwork.subtitle.physwindup": "Windin' up...",
+  "vs_clockwork.subtitle.boing": "Boioing!",
+  "vs_clockwork.subtitle.doink": "Doink.",
+  "alt_meter.trigger_height": "高度",
+  "sequenced_seat.operation.nothing": "没有",
+  "sequenced_seat.operation.distance": "移动",
+  "sequenced_seat.operation.angle": "旋转",
+  "sequenced_seat.operation.multiply": "倍增",
+  "sequenced_seat.value.distance": "距离",
+  "sequenced_seat.value.angle": "角度",
+  "sequenced_seat.value.multiply": "倍增器",
+  "vs_clockwork.subtitle.thick_fluid_swim": "在黏性流体中跋涉",
+  "vs_clockwork.subtitle.thick_fluid_fill": "充满了黏性流体",
+  "vs_clockwork.subtitle.thick_fluid_empty": "倒入黏性流体",
+  "vs_clockwork.subtitle.supersonic": "超级索尼克!",
+  "item.vs_clockwork.vanilla_frosting_bucket": "香草糖霜桶",
+  "item.vs_clockwork.chocolate_frosting_bucket": "巧克力糖霜桶",
+  "item.vs_clockwork.strawberry_frosting_bucket": "草莓糖霜桶",
+  "fluid.vs_clockwork.vanilla_frosting_bucket": "香草糖霜",
+  "fluid.vs_clockwork.chocolate_frosting_bucket": "巧克力糖霜",
+  "fluid.vs_clockwork.strawberry_frosting_bucket": "草莓糖霜",
+  "block.vs_clockwork.phys_bearing": "物理轴承",
+  "item.vs_clockwork.phys_bearing": "物理轴承",
+  "vs_clockwork.gravitron.tool.grab": "抓取",
+  "vs_clockwork.gravitron.tool.assemble": "组装",
+  "vs_clockwork.gravitron.tool.grab_and_assemble": "抓取并组装",
+  "vs_clockwork.gravitron.tool.destroy": "拆解",
+  "vs_clockwork.gui.toolmenu.focusKey": "按住 [%1$s] 以集中",
+  "vs_clockwork.gui.toolmenu.cycle": "[SCROLL] 到循环",
+  "vs_clockwork.gravitron.tool.grab.description.0": "抓住船.",
```

```suggestion
  "vs_clockwork.gravitron.tool.grab.description.0": "抓住船。",
```
这句话在什么地方会出现？原文`Grabs a ship.`

此外，标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

ship在此模组中指代物理结构，已修订用词